kubernetes/pkg-descr

```diff
@@ -0,0 +1 @@
+Production-Grade Container Scheduling and Management.
```

Comment:
```suggestion
Kubernetes, also known as K8s, is an open source system for automating
deployment, scaling, and management of containerized applications.  It
groups containers that make up an application into logical units for
easy management and discovery.
```
From <https://kubernetes.io/>. 

The description of a package must be longer than the comment. 

<https://docs.freebsd.org/en/books/porters-handbook/book/#porting-pkg-descr>